src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -122,6 +122,23 @@ public AcquireTokenForClientParameterBuilder WithPreferredAzureRegion(bool useAz
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// </summary>
+        public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
+        {
+            ValidateUseOfExperimentalFeature();
+
+            if (CommonParameters.CacheKeyComponents == null)
+            {
+                CommonParameters.CacheKeyComponents = new SortedList<string, string>();
+            }
+
+            CommonParameters.CacheKeyComponents.Add("fmi_path", pathSuffix);
```

Comment:
When the issue says "It will change the access token cache key by adding fmi_path to the cache key"
This is what you mean right?

Just want to make sure.